plone/app/locales/locales/nl/LC_MESSAGES/widgets.po

```diff
@@ -55,19 +59,19 @@ msgstr "Nieuw submap toevoegen in huidige map"
 
 #: ./patterns/filemanager/js/customize.js
 msgid "Add new override"
-msgstr ""
+msgstr "Nieuwe aanpassing toevoegen"
```

Comment:
1) Ik zou "Nieuwe" bij deze 4 vertalingen weglaten.   Overschrijving toevoegen, Pattern toevoegen, enz. 

2) aanpassing -> overschrijving

Reply:
Fixed directly on master in commit 856e7c975eeeac454d8dc057e661c96fdf3c11db.